locales/base/translation.json

```diff
@@ -2845,5 +2845,16 @@
     "ctaAction_Swap": "Swap smaller amount",
     "ctaAction_Deposit": "Deposit smaller amount",
     "ctaAction_Withdraw": "Withdraw smaller amount"
-  }
+  },
+  "reviewTransaction": {
+    "totalAmount_mutlipleTokens_noFiatPrice": "≈ {{amount1}} {{symbol1}} + {{amount2}} {{symbol2}}"
+  },
+  "tokenAmount": "{{tokenAmount}} {{tokenSymbol}}",
+  "tokenAmountApprox": "≈ {{tokenAmount}} {{tokenSymbol}}",
+  "localAmount": "{{localCurrencySymbol}}{{localAmount}}",
+  "localAmountApprox": "≈ {{localCurrencySymbol}}{{localAmount}}",
+  "tokenAndLocalAmount_oneToken": "{{tokenAmount}} {{tokenSymbol}} <0>({{localCurrencySymbol}}{{localAmount}})</0>",
+  "tokenAndLocalAmount_oneToken_noFiatPrice": "{{tokenAmount}} {{tokenSymbol}}",
+  "tokenAndLocalAmountApprox_oneToken": "≈ {{tokenAmount}} {{tokenSymbol}} <0>({{localCurrencySymbol}}{{localAmount}})</0>",
+  "tokenAndLocalAmountApprox_oneToken_noFiatPrice": "≈ {{tokenAmount}} {{tokenSymbol}}"
```

Comment:
nit: the translation keys that use the underscores are usually having the part after the underscore passed via [`context`](https://www.i18next.com/translation-function/context), but i don't think that is now they are being used here? i haven't given this a ton of thought but one option is to use the the total amount as the translation and use the context for the variants...e.g. transactionTotal_fiatOnly / transactionTotal_singleTokenWithFiat etc. ? alternatively just separate keys without the underscores?